packages/fetchai/protocols/fipa/dialogues.py

```diff
@@ -208,180 +202,171 @@ def add_dialogue_endstate(
 class FipaDialogues(Dialogues):
     """The FIPA dialogues class keeps track of all dialogues."""
 
-    def __init__(self) -> None:
+    def __init__(self, agent_address: Address) -> None:
         """
         Initialize dialogues.
 
         :return: None
         """
-        Dialogues.__init__(self)
-        self._initiated_dialogues = {}  # type: Dict[DialogueLabel, FipaDialogue]
-        self._dialogues_as_seller = {}  # type: Dict[DialogueLabel, FipaDialogue]
-        self._dialogues_as_buyer = {}  # type: Dict[DialogueLabel, FipaDialogue]
+        Dialogues.__init__(self, agent_address)
```

Comment:
`agent_address=agent_address` as we will make it a kwarg

Reply:
I changed it to that. But it there is no kwarg in the base constructor and I don't see why we might add it there.